Update ajax.php pour le PHP strict

tybox opened the pull request.

tybox committed:
Update ajax.php pour le PHP strict

tybox committed:
Merge branch 'simonvdfr:master' into master

tybox committed:
update envoi de mail

Renforcement du header du mail envoyé

simonvdfr commented:
Le code n'est pas idéal, car tu utilises $_REQUEST['email'] pour le from et repli-to, or ce mail peut avoir un domaine différent de celui du site (si la personne a comme mail de connexion une adresse gmail par exemple).
Pour un envoi de mail plus fiables (qui va moins dans les spam), il faut que l'adresse email qui envoie les mail depuis le site soit du même domaine potentiellement que d'où part le script... je recommanderai de plus mettre $GLOBALS['email_contact'] qui est dans config et qui normalement a la même extension que le domaine...
De plus, l'utilisateur qui reçoit le mail s'il veut contacter l'admin en retour du mail il ne peut pas car c'est son propre mail...

tybox committed:
Update ajax.php

Changement des données dans le header du mail d'envoi du mot de passe pour une meilleure cohérence.

tybox commented:
Je viens de faire le changement : effectivement c'est beaucoup plus cohérent avec ta suggestion.
